eventActions/checkChainMessage.js

```diff
@@ -36,7 +38,7 @@ class CheckChainMessage {
 		const messageContent = message.content;
 
 		// If the message length is greater than 20, forgettaboutit (potentially something editable in config?)
-		if(messageContent.length >= 20) {
+		if(messageContent.length >= config.chainMessageCharLimit) {
 			this.resetChannelChain(channelId);
```

Comment:
I don't fully understand, but `resetChannelChain` is required because if the length is greater than the limit, we know it is not the same as the currently stored message and therefore the chain has reset. i.e. you could have

User1: hello!
User2: hello!
User3: somereallylongmessage
USer4: hello!
Horace: hello!

I think that's what you're saying at least? The chain has to reset usually so return doesn't quite do all I need.